packages/modules/viewer-react/src/services/BaseInitializer.ts

```diff
@@ -71,6 +71,7 @@ const syncActiveSelectionScope = () => {
   Presentation.selection.scopes.activeScope =
     UiFramework.getActiveSelectionScope();
 
+  // eslint-disable-next-line deprecation/deprecation
```

Comment:
I think you can just remove the explicit type for `args` and it will be inferred, thus removing the need to use the deprecated `UiSyncEventArgs` type.

Reply:
Yep. Thanks again. Fixed.

Reply:
You left the `UiSyncEventArgs` import at the top of the file.